scipy/interpolate/_rbfinterp_pythran.py

```diff
@@ -173,7 +173,7 @@ def _build_system(y, d, smoothing, kernel, epsilon, powers):
 #                          float[:],
 #                          float[:, :])
 def _evaluate(x, y, kernel, epsilon, powers, shift, scale, coeffs):
```

Comment:
This function should be renamed and the doc strings should be updated since it no longer evaluates the interpolant. Perhaps rename it to `_build_evaluation_matrix`? Also, `coeffs` should be removed since it is no longer used.

Reply:
fixed in new version